src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -50,6 +54,7 @@ public class LanguageServiceAccessor implements Disposable {
     private static final Logger LOGGER = LoggerFactory.getLogger(LanguageServiceAccessor.class);
 
     private final Project project;
+    private final SimpleModificationTracker modificationTracker = new SimpleModificationTracker();

```

Comment:
This modification tracker allows us to evict when project-specific server statuses have changed and not all wrapper-level modification trackers are yet available. Once all servers have started, this modification is no longer added/used (see `getModificationTrackers()` below).